### PR TITLE
--dry-run

### PR DESCRIPTION
**Pull request by kentwills**
@asottile

### CODE REVIEW

File: paasta_tools/cli/cmds/local_run.py [hunk 1]
```diff
@@ -368,6 +369,14 @@ def add_subparser(subparsers):
...
         default=False,
     )
+    list_parser.add_argument(
+        '-d', '--dry-run',
+        help='Shows the arguments in json format supplied to docker',
+        action='store_true',
+        dest='dry_run',
+        required=False,
+        default=False,
+    )
```
Comment (asottile) [inline paasta_tools/cli/cmds/local_run.py:376]: default=False, required=False and dest=dry_run are the defaults

File: tests/cli/test_cmds_local_run.py [hunk 2]
```diff
@@ -435,6 +438,7 @@ def test_run_cook_image_fails(
     args.healthcheck = False
     args.interactive = False
     args.pull = False
+    args.dry_run = False
     assert paasta_local_run(args) is 1
```
Comment (asottile) [inline tests/cli/test_cmds_local_run.py]: ooh, not your code but they are asking for bugs here.  They should be using ==

**Comment by asottile** [comment]
add a test, otherwise looks good

**Comment by asottile** [comment]
capsys is probably your friend here if you're going to test this

**Comment by kentwills** [comment]
Working on the test

**Pull request closed without merging**